demo/paper/src/paper.js

```diff
@@ -4,6 +4,26 @@ var paper = new joint.dia.Paper({
     width: 600,
     height: 400,
     gridSize: 1,
+    drawGrid: {
+        //default grid - dot
+        color: 'black',
+        thickness: 1,
+
+        /**
+         * define custom grid pattern shape
+         */
+        // gridPattern: '<rect width="100%" height="100%" stroke="red" stroke-width="1" fill="transparent"/>',
+
+        /**
+         * define custom shape with updating attributes
+         */
+        gridPattern: '<path stroke="green" stroke-width="1"/>',
+        updateGridPattern: function(el, opt) {
```

Comment:
Please always expose `SVGElement|HTMLElement` not Vectorizer element nor jQuery element.